templates/web/common/site_layout.tt.html

```diff
@@ -367,6 +367,7 @@ <h3 class="title-5 text-medium">[% lang('footer_discover_the_project') %]</h3>
 							<li><a class="button small white-button radius" href="[% lang('footer_translators_link') %]">[% lang('footer_translators') %]</a></li>
 							<li><a class="button small white-button radius" href="[% lang('footer_partners_link') %]">[% lang('footer_partners') %]</a></li>
 							<li><a class="button small white-button radius" href="[% lang('footer_obf_link') %]">[% lang('footer_obf') %]</a></li>
+							<li><a class="button small white-button radius" href="[% lang('footer_pro_link') %]">[% lang('footer_pro') %]</a></li>
```

Comment:
We have a special template variable for the producer platform, already localized:
producers_platform_url

```suggestion
							<li><a class="button small white-button radius" href="[% producers_platform_url %]">[% lang('footer_pro') %]</a></li>
```

and we can remove the footer_pro_link from the .po files